openage/convert/slp.pyx

```diff
@@ -49,14 +49,22 @@ cdef struct boundary_def:
 
 # SLP pixels can be very special.
 cdef enum pixel_type:
-    color_standard       # standard pixel
-    color_shadow         # shadow pixel
-    color_transparent    # transparent pixel
-    color_player         # non-outline player color pixel
-    color_black          # black outline pixel
-    color_special_1      # player color outline pixel
-    color_special_2      # black outline pixel
-
+    color_standard      # standard pixel
+    color_shadow        # shadow pixel
+    color_shadow_v4     # shadow pixel in the 4.0X version
+    color_transparent   # transparent pixel
+    color_player        # non-outline player color pixel
+    color_player_v4     # non-outline player color pixel
+    color_black         # black outline pixel
+    color_special_1     # player color outline pixel
+    color_special_2     # black outline pixel
```

Comment:
why change the comment position? that obscures the diff a bit.

Reply:
I think Eclipse aligned the comments to tab width (4 spaces). In the git diff it's readable btw, but Github marks every changed line green somehow.